### ISSUE TITLE
Client Control via http/osc

### ISSUE THREAD

**Issue by joshnortheimer** [mumble-voip/mumble#5741]
### Context

move users via ethernet control

### Description

I use mumble in a video production environment to talk to camera operators. we also use a program from Bitfocus, called Companion, which interfaces with all of our production gear, and allows control via an Elgato Streamdeck. I would like to be able to connect individual endpoints, or to the server using the superuser account, and be able to control functions such as moving users to a specific channel, or muting. right now, i'm controlling those endpoints using keyboard shortcuts, and a program called "VICREO listener" to push keyboard buttons via the network, but local keypresses while using other apps can mess things up. 

Basically, I'd like to push a button on my streamdeck, which connects to the server or clients over the network, and completes the same function as logging in to the software as superuser, and executes the same commands as keyboard shortcuts, but without the keyboard. perhaps using OSC commands, or a http commands.

it could be as simple as: [serverIpAddress]:\\[user]\move\\[newChannelAssignment]

Thanks!!

### Mumble component

Both

### OS-specific?

No

**Comment by Krzmbrzl** [comment]
It sounds like what you want could (and probably should) be done by a plugin. There already exists a proof of concept for a Stream Deck Mumble interface that allows you to remote control a Mumble instance via the Stream Deck. 
I guess this could be extended to fit your need.

An alternative would be for you to use the ICE interface to the server which can also do such administrative tasks. Then you don't even need a Mumble client at all.
